### ISSUE TITLE
psycopg2.ProgrammingError: can't adapt type 'numpy.int64' DO SOON

### ISSUE THREAD

**Issue by pjcpjc** [ticdat/ticdat#68]
Pretty easy to create with `testing.postgresql ` by just having some data. This snippet created the bug.

```
import testing.postgresql as testing_postgresql
import sqlalchemy as sa
postgresql = testing_postgresql.Postgresql()
engine = sa.create_engine(postgresql.url())
import diet
dat = diet.input_schema.csv.create_tic_dat("/Users/petercacioppi/src/ticdat/examples/gurobipy/diet/diet_sample_data/")
schema = "test"
diet.input_schema.pgsql.write_schema(engine, schema, include_ancillary_info=False)
import numpy
for r in dat.nutrition_quantities.values():
    if int(r["Quantity"]) == r["Quantity"]:
       r["Quantity"] = numpy.int64(r["Quantity"])
diet.input_schema.pgsql.write_data(dat, engine, schema, dsn=postgresql.dsn())
```

**Comment by pjcpjc** [comment]
And then this fixed it.

```
diet.input_schema.pgsql.write_data(dat, engine, schema, dsn=postgresql.dsn())
from hackit import make_all_numbers_floats
dat_2 = make_all_numbers_floats(diet.input_schema, dat)
diet.input_schema.pgsql.write_data(dat_2, engine, schema, dsn=postgresql.dsn())
dat_3 = diet.input_schema.pgsql.create_tic_dat(engine, schema)
diet.input_schema._same_data(dat, dat_3)
```

Where `hackit.py` was

```
scenario_id=f"scenario_{i}",
	app_id="5e7e842603bddb00231941a5",
	enframe_url="https://apps3.opexanalytics.com/",
	is_on_enframe=False,
	force_token_refresh=False
```

**Comment by pjcpjc** [comment]
Fixed in 9b5b4dc, 620aa15 was just a sanity double check
